PointOfDispensingApp/src/components/PatientLookupPage.vue

```diff
@@ -130,24 +128,65 @@
         this.selectedFamilyName=item.familyName
         this.selectedGivenName=item.givenName
         this.selectedDOB=item.DOB
+        this.selectedLineAddress=item.lineAddress
+        this.selectedCity=item.city
+        this.selectedPostalCode=item.postalCode
       },
       retrievePatientRecord()
       {
-        alert(this.selectedFamilyName + ', ' + this.selectedGivenName + ', ' + this.selectedDOB)
+        // To do:  call API to fetch patient record
+        // Once returned, call the function below (may need to pass in the payload):
+        this.patientRecordRetrieved()
+      },
+      patientRecordRetrieved()
+      {
+        //the following is sending dummy data until we have the API in place
+        const patientResourcePayload = {
+          patientId:'1234567890',
+          patientLastName: this.selectedFamilyName,
+          patientFirstName: this.selectedGivenName,
+          patientDateOfBirth: this.selectedDOB,
+          patientGender: 'Male',
+          patientStreetAddress: this.selectedLineAddress + ', ' + this.selectedCity + ", WI, " + this.selectedPostalCode,
+          patientPreferredLanguage: 'English'
+        }
+        //send data to Vuex
+        this.$store.dispatch('patientRecordRetrieved', patientResourcePayload)
+
+        //Advance to the Check In page
+        this.$router.push("CheckIn")
+      },
+      scanQrCode()
+      {
+        //the following is sending dummy data until we have the API in place
+        const patientResourcePayload = {
+          patientId:'1234567890',
+          patientLastName: 'Fraley',
+          patientFirstName: 'David',
+          patientDateOfBirth: '01/01/1950',
+          patientGender: 'Male',
+          patientStreetAddress: '1234 Main Street, Waukesha, WI, 53072',
+          patientPreferredLanguage: 'English'
+        }
+        //send data to Vuex
+        this.$store.dispatch('patientRecordRetrieved', patientResourcePayload)
+
+        //Advance to the Check In page
+        this.$router.push("CheckIn")
       }
     },
     components: 
     {
     },
     data () {
 	return {
-		patientFamilyName: '',
-		patientGivenName: '',
-    patientDate: '',
-    selectedId: -1,
+		selectedId: -1,
```

Comment:
-1? Why is that?

Reply:
It's initializing the ID of the selected row to be something that is obviously invalid.

Reply:
Can you add a comment clarifying that?

Reply:
Yep!